### PR TITLE
Story/quad 57

### PR DESCRIPTION
**Pull request by JNagarkar**
Added a ADD WORKSPACE button in project description and removed the workspace jstree list from MENU.

### CODE REVIEW

**Comment by jdamerow** [comment]
ok to test

File: Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp [hunk 1]
```diff
@@ -67,16 +67,24 @@
...
 					<a
 					href="${pageContext.servletContext.contextPath}/auth/workbench/workspace/workspacedetails/${workspace.workspaceId}"><c:out
 							value="${workspace.workspaceName}"></c:out></a> (Owner) <br> 
-						
+
+			
 					<c:out
 					value="${workspace.description}"></c:out>
 				</li>
+				 	<li data-jstree='{"icon":"${pageContext.servletContext.contextPath}/resources/txt-layout/css/images/down.png"}'>Workspace
```
Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp:85]: Why is this here?

Comment (jdamerow) [inline Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp:85]: Looks like an artifact.

Comment (JNagarkar) [inline Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp:85]: removed the artefact .
